front/pages/api/w/[wId]/assistant/conversations/index.ts

```diff
@@ -150,7 +185,26 @@ async function handler(
         newMessage = messageRes.value;
       }
 
-      res.status(200).json({ conversation, message: newMessage ?? undefined });
+      if (newContentFragment || newMessage) {
+        // If we created a user message or a content fragment (or both) we retrieve the
```

Comment:
Within the product we don't need an updated view, it will do so on its own. So we can save that get